### PR TITLE
os/board/rtl8720e: update wifi for August release

### PR DESCRIPTION
**Pull request by gexuyan**
* this kr4_image2_all.bin is based on roy_zhu's 121f48f9 commit on 8/8/2022 and has enabled BT configs

### CODE REVIEW

**Comment by sunghan-chang** [comment]
@gexuyan Could you let us know what this is briefly?
@j11591 After getting description, could you check merging?

**Comment by gexuyan** [comment]
> @gexuyan Could you let us know what this is briefly? @j11591 After getting description, could you check merging?

Dear @sunghan-chang,
This PR is to update kr4 image and km4 source code based on our freertos sdk, there're many new updates/bug fixes between this version and previous version on Tizenrt. This update is a pre-requisite for upcoming BT release which will be done by roy_zhu. Thank you~

**Review by j11591 — APPROVED** [review]
LGTM

File: os/board/rtl8720e/src/component/wifi/api/wifi_conf_inic_ipc.c [hunk 1]
```diff
@@ -443,7 +444,7 @@ int wifi_on(rtw_mode_t mode)
 		LwIP_netif_set_up(0);
 		if (mode == RTW_MODE_AP) {
 			LwIP_netif_set_link_up(0);
-		} else if (mode == RTW_MODE_STA_AP) {
+		} else	 if (mode == RTW_MODE_STA_AP) {
```
Comment (amogh-samsung) [inline os/board/rtl8720e/src/component/wifi/api/wifi_conf_inic_ipc.c:447]: Why this extra space between "else" and "if"?

File: os/board/rtl8720e/src/component/wifi/api/wifi_conf_inter.c [hunk 2]
```diff
@@ -0,0 +1,63 @@
+//----------------------------------------------------------------------------//
```
Comment (amogh-samsung) [inline os/board/rtl8720e/src/component/wifi/api/wifi_conf_inter.c:1]: We'll need an open source license here please.

File: os/board/rtl8720e/src/component/wifi/api/wifi_conf_inter_inic_ipc.c [hunk 3]
```diff
@@ -0,0 +1,73 @@
+
```
Comment (amogh-samsung) [inline os/board/rtl8720e/src/component/wifi/api/wifi_conf_inter_inic_ipc.c:1]: We'll need an open source license here?

File: os/board/rtl8720e/src/component/wifi/api/wifi_conf_inter.c (hunk 2 above)
Comment (gexuyan) [reply os/board/rtl8720e/src/component/wifi/api/wifi_conf_inter.c:1]: Dear @amogh-samsung,
I checked the other files in this layer, seems all don't provide license. These are the API files from our sdk, so maybe don't need license here. Is that okay for customer side?
Thank You.

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**